deploy/charts/hcloud-csi/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: csi-hcloud
```

Comment:
```suggestion
name: csi-driver
```

Reply:
Max compatibility with our existing manifests would be `hcloud-csi`